core/src/com/unciv/logic/map/MapUnit.kt

```diff
@@ -170,6 +170,8 @@ class MapUnit {
     var religion: String? = null
     var religiousStrengthLost = 0
 
+    var isLinked = false
```

Comment:
`linkedTo: MapUnit?`?

Reply:
Does it matter if we list what it is linked to? we can always call the function "getUnitNotOfType" to get the linkedunit.

Reply:
Interdice is correct here - we want to preserve linking between turns, so we need to save the link in a way that's json serializable